recipes/pygridgen/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+    name: pygridgen
+    version: 0.1.1
+
+source:
+    git_url: https://github.com/phobson/pygridgen.git
+    git_tag: 0.1.1
+
+build:
+    number: 0
+    skip: True  # [win]
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - numpy
+    run:
+        - python
+        - pandas
+        - numpy
+        - matplotlib
+        - pyproj
+        - gridgen
+
+test:
+    imports:
+        - pygridgen
+
+about:
+    home: https://github.com/hetland/pygridgen.git
+    license:  BSD License
```

Comment:
Initially, I was just meaning what BSD flavor, but I guess this is more complicated.

I opened this issue ( https://github.com/hetland/pygridgen/issues/27 ) to see if he will clarify. I am leaning towards saying MIT as it is referred to in a couple of places [here]( https://github.com/hetland/pygridgen/blob/9917ab351385a2311e286452815da3fe148b8234/setup.py#L13 ) and [here]( https://github.com/hetland/pygridgen/blob/9917ab351385a2311e286452815da3fe148b8234/pygridgen/__init__.py#L11 ). That [BSD]( https://github.com/hetland/pygridgen/blob/9917ab351385a2311e286452815da3fe148b8234/setup.py#L34 ) could have been leftover. Though we could say BSD or MIT and leave a note here to check on the issue above and update appropriately once clarified.